tools/devops/provision-localization.csx.in

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
why does the path have a .csx extension.

Reply:
I was just following the other examples. Should I give this the .json.in extension?

Reply:
of course, the other examples are using csx because they are csx files.